themes/bootstrap3/templates/record/addtag.phtml

```diff
@@ -20,4 +20,4 @@
   <div class="form-group">
     <input class="btn btn-primary" type="submit" value="<?=$this->transEscAttr('Save')?>"/>
   </div>
-</form
+</form>
```

Comment:
This is clearly a bug fix -- I've just picked this one change directly to the dev branch.

Reply:
Thanks! I was supposed to do that myself, but forgot.